src/components/App.js

```diff
@@ -6,49 +6,36 @@ import {
 } from 'react-router-dom';
 
 import { connect } from 'react-redux';
-import { createPost } from '../store/actions/postsActions';
+import { createPost, deletePost } from '../store/actions/postsActions';
 import { firestoreConnect } from 'react-redux-firebase';
 import { compose } from 'redux';
 import 'bulma/css/bulma.css';
 import '../styles/main.scss';
 
 import Navbar from './Nav/Navbar';
 import CreatePost from './CreatePost/CreatePost';
+import PostList from './PostList/PostList';
+import Post from './PostList/Post';
 import Profile from './Profile/Profile';
 import About from './About/About';
 
 class App extends Component {
-    render() {
+
+  render() {
     // console.log(this.props.posts[0])
     return (
       <Router>
         <div>
           <Navbar />
 
           <Switch>
-            <Route path='/about'>
-              <About />
-            </Route>
-            <Route path='/profile'>
-              <Profile />
-            </Route>
-            <Route path='/create-post'>
-              <CreatePost />
-            </Route>
+            <Route exact path='/' component={PostList} />
+            <Route path='/about' component={About} />
+            <Route path='/profile'component={Profile} />
+            <Route path='/create-post' component={CreatePost} />
+            <Route exact path='/posts/' component={PostList} />
+            <Route path='/posts/:id' component={Post} />
```

Comment:
With react-router-dom v5 the supposed preferred method is what was in place before, but this looks more pleasing to the eye, honestly.